<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OpenedListItem</class>
 <widget class="QWidget" name="OpenedListItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>52</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: rgba(0, 0, 0, 0.4);</string>
  </property>
  <widget class="QLabel" name="label_Cover">
   <property name="geometry">
    <rect>
     <x>24</x>
     <y>10</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">/* recent open item */


position: static;
width: 320px;
height: 52px;
left: 0px;
top: 32px;


/* Inside Auto Layout */

flex: none;
order: 0;
flex-grow: 0;
margin: 0px 0px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/SVG/svg/icon/AudiobookEditorLogo@16.svg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_Title">
   <property name="geometry">
    <rect>
     <x>68</x>
     <y>8</y>
     <width>144</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">position: absolute;
width: 144px;
height: 21px;
left: 0px;
top: 0px;

font-family: Noto Sans TC;
font-style: normal;
font-weight: normal;
font-size: 14px;
line-height: 21px;
/* identical to box height */


color: #FFFFFF;
background: rgba(0, 0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Approach</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Path">
   <property name="geometry">
    <rect>
     <x>68</x>
     <y>29</y>
     <width>228</width>
     <height>15</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">position: static;
width: 228px;
height: 15px;
left: 0px;
top: 29px;

font-family: Noto Sans TC;
font-style: normal;
font-weight: normal;
font-size: 11px;
line-height: 15px;
/* identical to box height */

background: rgba(0, 0, 0, 0);

color: rgba(255, 255, 255, 0.5);


/* Inside Auto Layout */

flex: none;
order: 1;
align-self: stretch;
flex-grow: 0;
margin: 0px 0px;</string>
   </property>
   <property name="text">
    <string>~/Desktop/Audiobook-Demo/test-12/Approach...</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Ago">
   <property name="geometry">
    <rect>
     <x>232</x>
     <y>8</y>
     <width>64</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans TC</family>
     <pointsize>-1</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">position: absolute;
width: 64px;
height: 21px;
left: 164px;
top: 0px;

font-family: Noto Sans TC;
font-style: normal;
font-weight: normal;
font-size: 11px;
line-height: 16px;
display: flex;
align-items: center;
text-align: right;

color: #7EC1FF;

background: rgba(0, 0, 0, 0);</string>
   </property>
   <property name="text">
    <string>3 days ago</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
